test/support.js

```diff
@@ -19,6 +19,10 @@ expectFileExists = function(file) {
   assert(fs.existsSync(file), file + ' does not exist');
 };
 
+expectFileNotExists = function(file) {
+  assert(!fs.existsSync(file), file + ' does exist');
```

Comment:
existsSync is deprecated in new node. Should use fs.statSync or fs.accessSync instead: see https://nodejs.org/api/fs.html#fs_fs_existssync_path